[2024-12-07T20:38:55.442+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.Is_weather_api_ready manual__2024-12-07T20:38:53.761411+00:00 [queued]>
[2024-12-07T20:38:55.448+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.Is_weather_api_ready manual__2024-12-07T20:38:53.761411+00:00 [queued]>
[2024-12-07T20:38:55.449+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-12-07T20:38:55.457+0000] {taskinstance.py:2191} INFO - Executing <Task(HttpSensor): Is_weather_api_ready> on 2024-12-07 20:38:53.761411+00:00
[2024-12-07T20:38:55.462+0000] {standard_task_runner.py:60} INFO - Started process 15870 to run task
[2024-12-07T20:38:55.465+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'Is_weather_api_ready', 'manual__2024-12-07T20:38:53.761411+00:00', '--job-id', '346', '--raw', '--subdir', 'DAGS_FOLDER/extract.py', '--cfg-path', '/tmp/tmpul678j_r']
[2024-12-07T20:38:55.465+0000] {standard_task_runner.py:88} INFO - Job 346: Subtask Is_weather_api_ready
[2024-12-07T20:38:55.477+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-12-07T20:38:55.505+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.Is_weather_api_ready manual__2024-12-07T20:38:53.761411+00:00 [running]> on host 8271fbcad3f4
[2024-12-07T20:38:55.587+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='thiernosidybah232@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='Is_weather_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2024-12-07T20:38:53.761411+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-07T20:38:53.761411+00:00'
[2024-12-07T20:38:55.588+0000] {http.py:130} INFO - Poking: /data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-07T20:38:55.599+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-07T20:38:55.970+0000] {base.py:295} INFO - Success criteria met. Exiting.
[2024-12-07T20:38:55.977+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=Is_weather_api_ready, execution_date=20241207T203853, start_date=20241207T203855, end_date=20241207T203855
[2024-12-07T20:38:55.997+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-12-07T20:38:56.037+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
